app/assets/javascripts/comfy/admin/cms/filters.js.coffee

```diff
@@ -0,0 +1,14 @@
+$(document).ready( ->
+  # Create a change event listener for each topic checkbox.
+  # This will add or remove the related substring in the content input tag.
+  ['WDPA', 'OECM', 'PAME'].forEach((el) =>
+    $('#fragment-topic_'+el).change((event) =>
```

Comment:
Where does `#fragment-topic` come from? Doesn't seem to be defined in `filter_checkboxes`, so I assume it's somehow being incorporated into the ID of the element.